sprint9/tema1.js

```diff
@@ -0,0 +1,26 @@
+// am creat o functie pentru a crea un obiect de tip reteta, avand 3 parametrii
+function creeazaReteta(titlu, portii, ingrediente){
+    const reteta = {
+ // primul parametru e titlu si anume un sir de caractere, unde avem numele reteti
```

Comment:
Ar fi bine si comentariile sa respecte identarea, in caz contrar citirea codului este complicat